src/contract.jl

```diff
@@ -10,10 +10,45 @@ function contract(
 end
 
 function contract(
-  alg::Union{Algorithm"density_matrix",Algorithm"ttn_svd"},
+  alg::Union{Algorithm"density_matrix",Algorithm"density_matrix_direct_eigen",Algorithm"ttn_svd"},
   tn::AbstractITensorNetwork;
   output_structure::Function=path_graph_structure,
   kwargs...,
 )
   return approx_itensornetwork(alg, tn, output_structure; kwargs...)
 end
+
+function contract(
+  alg::Algorithm"partitioned_contract",
+  tn::AbstractITensorNetwork;
+  nvertices_per_partition::Integer=2,
+  backend::String="KaHyPar",
+  kwargs...,
+)
+  sequence = _partitioned_contraction_sequence(
+    tn; nvertices_per_partition=nvertices_per_partition, backend=backend
```

Comment:
```suggestion
    tn; nvertices_per_partition, backend
```
In the more recent versions of Julia, you don't have to repeat the argument if it has the same name as the keyword in a keyword argument.